src/monitoring/bin/db_full_check.sh

```diff
@@ -0,0 +1,8 @@
+echo CROMWELL_GOOGLE_PROJECT is ${CROMWELL_GOOGLE_PROJECT}
```

Comment:
To error out if there are not set, should we have `set -euo pipefail`?

Reply:
The wrapper does that already but I could do it here too.